### PR TITLE
Add the maintenors

### PR DESCRIPTION
**Pull request by bkhelifi**
I have added a section on the modules maintenors in the team page.
It contains their functions and the list of maintenors...

Signed-off-by: Bruno Khélifi <khelifi@in2p3.fr>

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
             The current lead developers are <strong>Axel Donath</strong> and <strong>Régis Terrier</strong>.
         </p>
 
+        <h2 id="module-maintainer">Module maintainers</h2>
+          <p>Amongst the development team, some experts are devoted to the maintenance of some modules.</p>
+          
+          <p>They are in charge of:
+          <ul>
+            <li>supporting developers on tasks associated to their module(s),</li>
+            <li>making a review of proposed contributions for their module(s),</li>
```
Comment (registerrier) [inline team.html:294]: Reviewing contributions impacting the module

File: team.html [hunk 2]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
             The current lead developers are <strong>Axel Donath</strong> and <strong>Régis Terrier</strong>.
         </p>
 
+        <h2 id="module-maintainer">Module maintainers</h2>
+          <p>Amongst the development team, some experts are devoted to the maintenance of some modules.</p>
+          
+          <p>They are in charge of:
+          <ul>
+            <li>supporting developers on tasks associated to their module(s),</li>
```
Comment (registerrier) [inline team.html:293]: I would avoid writing 'their module' as no part actually belongs to anyone

File: team.html (hunk 1 above)
Comment (registerrier) [reply team.html:294]: Or following the astropy team description:
Evaluating new pull requests for quality, API consistency and Gammapy coding standards

File: team.html [hunk 3]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
...
+            <li>making a review of proposed contributions for their module(s),</li>
+            <li>taking care of the global design of their module(s) in the context of the global Gammapy architecture,</li>
+            <li>participating to the User Support for questions related to their module(s).</li>
```
Comment (registerrier) [inline team.html:296]: I would add:
perform initial triage of issues and pull requests

File: team.html [hunk 4]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
...
+
+          <p> The list of maintainers is given below:
+              <table>
+                  <tr>
+                      <th>Maintainer</th>
+                      <th>Module(s)</th>
+                  </tr>
+                  <tr>
+                      <td>Quentin Rémy (MPIK)</td>
+                      <td><a href="https://docs.gammapy.org/dev/api-reference/datasets.html">datasets</a>,
```
Comment (registerrier) [inline team.html:308]: Maybe give module names in alphabetical order

File: team.html [hunk 5]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
             The current lead developers are <strong>Axel Donath</strong> and <strong>Régis Terrier</strong>.
         </p>
 
+        <h2 id="module-maintainer">Module maintainers</h2>
+          <p>Amongst the development team, some experts are devoted to the maintenance of some modules.</p>
```
Comment (registerrier) [inline team.html:289]: Among the Gammapy core developer team, they are

Comment (registerrier) [reply team.html:289]: Maybe some word on how they are chosen? Do we validate the choice by the CC?

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi ! Some inline comments.
Is the choice to be validated by the CC?

Any feedback on the PR @adonath ?

File: team.html [hunk 6]
```diff
@@ -103,6 +124,7 @@ <h2 id="overview">Overview</h2>
             <li><a href="#coordination-committee">Coordination committee</a></li>
             <li><a href="#project-manager">Project managers</a></li>
             <li><a href="#lead-developer">Lead developers</a></li>
+            <li><a href="#module-maintainer">Module maintainers</a></li>
```
Comment (adonath) [inline team.html:127]: Would call it "Sub-package maintainer", which I think is consistent with Astropy's definition...

File: team.html [hunk 7]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
...
+          <p>They are in charge of:
+          <ul>
+            <li>supporting developers on tasks associated to their module(s),</li>
+            <li>making a review of proposed contributions for their module(s),</li>
+            <li>taking care of the global design of their module(s) in the context of the global Gammapy architecture,</li>
+            <li>participating to the User Support for questions related to their module(s).</li>
+          </ul>
+          </p>
+
+          <p> The list of maintainers is given below:
```
Comment (adonath) [inline team.html:300]: sub-package maintainers

File: team.html [hunk 8]
```diff
@@ -263,6 +285,40 @@ <h2 id="lead-developer">Lead developers</h2>
...
+              <table>
+                  <tr>
+                      <th>Maintainer</th>
```
Comment (adonath) [inline team.html:303]: Sub-package maintainer

**Review by adonath — COMMENTED** [review]
Thanks @bkhelifi! I think would try to consistently use "sub-package maintainer" as a role name...

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi !

**Pull request merged**